jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -117,6 +117,7 @@ with what features/services are supported.
         </extensionPoint>
 
         <extensionPoint name="executable" interface="software.aws.toolkits.jetbrains.core.executables.ExecutableType" dynamic="true"/>
+        <extensionPoint name="executable2" interface="software.aws.toolkits.jetbrains.core.executables.ExecutableType2" dynamic="true"/>
```

Comment:
Technically not needed anymore, ExecutableInstances2's should all be objects and stateless

TODO: Delete this...